src/transformers/configuration_utils.py

```diff
@@ -635,7 +635,7 @@ def _get_config_dict(
         force_download = kwargs.pop("force_download", False)
         resume_download = kwargs.pop("resume_download", False)
         proxies = kwargs.pop("proxies", None)
-        use_auth_token = kwargs.pop("use_auth_token", None)
+        token = kwargs.pop("token", None)
```

Comment:
This is `_get_config_dict`. We shouldn't need to worry a user's direct call to this with `use_auth_token`?